tasks/main.yml

```diff
@@ -3,19 +3,17 @@
 # Prepare system
 - ansible.builtin.include: 'prepare_user.yml'
 
-# this recursively creates configuration directory with root:root 755 permissions
-- name: Ensure configuration directory exists
+- name: Create configuration directory parent
   ansible.builtin.file:
     state: directory
-    path: "{{ item }}"
+    path: "{{ item | realpath | dirname }}"
```

Comment:
Let's move `realpath | dirname` filter to the `"{{ neofs_storage__conf_dir }}"` expression

Reply:
Nope, it's intended. This filter is common for all `items`.